### DISCUSSION TITLE
gitbash update problem

### DISCUSSION THREAD

**Discussion by Kc9jjj** [community/community#66963] [Programming Help]
### Select Topic Area

Bug

### Feature Area

Issues

### Body

when I try to update the submodules of the repo i'm working on I get the following error 
$ git submodule update --init --recursive                                                                                                                                          
fatal: transport 'file' not allowed                                                                                                                                                
fatal: Fetched in submodule path 'include/nvda-cldr', but it did not contain e3af0878691391cc019d2e1e7965aa5756ac653a. Direct fetching of that commit failed.                      
 this is on git 2.4 on windows 11 if I run the command on ubuntu I don't get this error.

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @Kc9jjj ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Projects and Issues category is solely related to conversations around the GitHub products Issues and Projects.

We’ve moved your post to our new Programming Help 🧑‍💻 category, which is more appropriate for this type of discussion.

Please review our [guidelines](https://github.com/orgs/community/discussions/51384) about the Programming Help category for more information.
